### ISSUE TITLE
Bug - : Website: Extremely large icons

### ISSUE THREAD

**Issue by eltonbautista** [TheOdinProject/theodinproject#2664]

**Comment by dm-murphy** [comment]
This issue has been transferred to the main site repo

**Comment by xandora** [comment]
I think this was supposed to be fixed in #2642 ?

**Comment by KevinMulhern** [comment]
Thanks for reporting @eltonbautista, and thanks for including the video, it is immensely helpful.

@xandora I think those changes might have only reduced the frequency of it unfortunately 😓 

I believe the issue is that we are loading two stylesheets and one of them is being blocked, 
* The first stylesheet is through the asset pipeline for our older sass and bootstrap CSS.
* The second is through webpacker and loads in the newer Tailwind styles we use.

It seems like the second stylesheet is sometimes blocked from loading on page load. You can see it in the video, the top of the page has no styling and we recently switched the navbar over to Tailwind. When they scroll down and reach the lesson content, the styles come back again. That content is still using our old CSS.

**Comment by eltonbautista** [comment]
I just noticed that 2 parts of the I sent were commented out. I have updated the original post, my comments for "How to Reproduce" as well as "Expected Behavior" are now visible if that might help you guys out solving it. 
Note: I've never experienced it when loading onto the site by clicking "next lesson" or just going directly to a lesson from the curriculum list. This bug only happens when I restore session or use ctrl+shift+t to reopen old tabs - as described in my original comment.

**Comment by bappyasif** [comment]
just a note: im also facing this 'extremely enlarged icons' issue when i'm 'opening chrome, with "start where i left off" option enabled', and i also asked this in discord as well [message link](https://discord.com/channels/505093832157691914/540903304046182425/956523529191448627),  thanks to leo for introducing me to this issue initially, much appreciated....

![image](https://user-images.githubusercontent.com/19854521/159988666-6958d3c2-233e-4a49-98d6-f5ca36ef740b.png)

**Comment by bappyasif** [comment]
@KevinMulhern is this issue solved, cause i'm not seeing this from 'initial browser start view-with where left off option'

**Comment by KevinMulhern** [comment]
I don't think so @bappyasif, we haven't released anything recently which could resolve it.

**Comment by bappyasif** [comment]
well, if that makes it any better or worse, im not seeing that on my end from that scenario, where i start chrome, with "start where i left off", option enabled, is no longer showing me 'extremely large' icons which it was used to. just so you know, and happy coding :) @KevinMulhern

**Comment by KevinMulhern** [comment]
I haven't heard of this one occurring again since the upgrade to Tailwind 3. Closing.
